### PR TITLE
doc: updated OpenStack rbd documentation

### PR DESCRIPTION
**Pull request by falcon78921**
- [x] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

Removed references to older OpenStack releases. Updated
documentation in regards to using `.qcow2` as a virtual machine
disk type.

Fixes: https://tracker.ceph.com/issues/18197
Signed-off-by: James McClune <jmcclune@mcclunetechnologies.net>

### CODE REVIEW

File: doc/rbd/rbd-openstack.rst [hunk 1]
```diff
@@ -252,15 +219,7 @@ Any OpenStack version except Mitaka
...
-    show_image_direct_url = True
-
-For Mitaka only
-^^^^^^^^^^^^^^^
-
-To enable image locations and take advantage of copy-on-write cloning for images, add under the ``[DEFAULT]`` section::
-
-    show_multiple_locations = True
-    show_image_direct_url = True
+show_image_direct_url = True
```
Comment (dillaman) [inline doc/rbd/rbd-openstack.rst]: Nit: missing indentation causing doc build error:

> /home/jenkins-build/build/workspace/ceph-pr-docs/doc/rbd/rbd-openstack.rst:222: WARNING: Literal block expected; none found.

**Comment by falcon78921** [comment]
jenkins retest this please

File: doc/rbd/rbd-openstack.rst (hunk 1 above)
Comment (falcon78921) [reply doc/rbd/rbd-openstack.rst]: @dillaman I made the change :)

**Comment by dillaman** [comment]
jenkins render docs

**Pull request merged**